Documentation/README_ProgressIndicator.md

```diff
@@ -0,0 +1,72 @@
+# Progress Indicators
+## Example scene
```

Comment:
Can you actually start with a small section on what progress indicators are and the types auto-included with MRTK. Maybe pull up bits about the IProgressIndicator interface up here

Reply:
Other minor note. In general, guidelines are to avoid use of informal you/we

Reply:
I know this isn't followed everywhere unfortunately for consistency

Reply:
I left out the number of indicators we include / any details about specific indicators purposefully so when we add more we won't have to update the docs. The demo scene does a better job of documenting them IMO.